omnibus/config/software/confluent-kafka-python.rb

```diff
@@ -0,0 +1,38 @@
+# https://github.com/confluentinc/confluent-kafka-python/blob/master/INSTALL.md#install-from-source
+
+name "confluent-kafka-python"
+default_version "2.0.2"
+
+dependency "pip3"
+dependency "librdkafka"
+
+source :url => "https://github.com/confluentinc/confluent-kafka-python/archive/refs/tags/v#{version}.tar.gz",
+       :sha256 => "137c82d23e931e03a2803569d51bb025a7b9a364818b08d1664800abaaa5cc63",
+       :extract => :seven_zip
+
+relative_path "confluent-kafka-python-#{version}"
+
+build do
+  license "Apache-2.0"
+  license_file "./LICENSE.txt"
+
+  if windows?
+    librd_dir = "c:\\librdkafka-redist"
+    build_env = {
+        "INCLUDE" => "#{librd_dir}\\librdkafka.redist.#{version}\\build\\native\\include",
+        "LIB" => "#{librd_dir}\\librdkafka.redist.#{version}\\build\\native\\lib\\win\\x64\\win-x64-Release\\v142"
+    }
```

Comment:
The OOTB confluent-kafka wheels already provides librdkafka (but whitout kerberos support ofc) so we can simply `pip install` it and get rid of everything else. You can also move the `dependency "librdkafka"` to the else and drop the windows part from the `librdkafka.rb` file

Reply:
Ah yeah I added this back in my commit changing the `pip` command to see if the `INCLUDE` or `LIB` flags were being used in some way since the CI was originally failing. I'll apply your suggestions now since the CI is passing!

Reply:
Applied: https://github.com/DataDog/datadog-agent/pull/15470/commits/355b11a31a93832759c94de8e3d626c94c23edef